### DISCUSSION TITLE
No Preview Cards on our server - why?

### DISCUSSION THREAD

**Discussion by stephan0h** [mastodon/mastodon#29485] [Q&A]
On our server links are just displayed as links. On other servers nice preview cards are rendered. We are on 4.2.7 - so an old version can be ruled out to cause the problem. We cleaned our cache (which was quite full) - but still no turn for the better. What else should we check?
Hints are much appreciated!
Kind regards,
Stephan

**Comment by mjankowski** [comment]
Did you ever sort this out?

If not, can you confirm what version the server is on now and that you still see the issue?

Also just to check - do you see this issue for *the same links* that seem to work as preview cards on other serveers?
